packages/documentation/src/pages/platform/front-end-standards/manual-reviews.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Manual reviews
+tags: vagovprod, vagovstaging, vagovdev
+---
+
+# Manual reviews
+
+Each pull request (PR) on `vets-website` runs through an automated process looking for code additions and modifications that don't meet code quality standards. If a potential issue is found, a bot will leave an automated comment and request a review from the **frontend-review-group** (see below).
+
+   ![Screen Shot](https://images.zenhubusercontent.com/5e387fa505ac7e30c820a2da/a09c4c8c-ae32-4142-8fa5-0fc81b6a1892)
+
+The following items will trigger a manual review:
+
+- A call to `Sentry`
+- ESLint being disabled
+- An icon is added with `<i ...`
+
+### Sentry calls
+
+VSP reviews logs to Sentry to ensure that any new logs are useful and do not contain personally identifiable information (PII).
+
+Examples:
+- Do not log an entire request response if all that's important is an error code.
+- Do not log PII such as names, emails, or user IDs.
+
+### ESLint
+
+VSP enforces [ESLint rules](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/engineering/frontend/eslint) through CircleCI to enforce code quality. ESLint rules are included in the `.eslintrc` file, which is located in the root folder. Any rules that VSP is [testing for inclusion](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/frontend/eslint/adding-new-rules.md) are located in the `circle.eslint.json` file.
```

Comment:
`circle.eslint.json` lives in the `vets-website` repo:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/circle.eslint.json

Reply:
Added link